sdk/provisioning/Azure.Provisioning.AppConfiguration/README.md

```diff
@@ -22,6 +22,52 @@ dotnet add package Azure.Provisioning.AppConfiguration
 
 This library allows you to specify your infrastructure in a declarative style using dotnet.  You can then use azd to deploy your infrastructure to Azure directly without needing to write or maintain bicep or arm templates.
 
+## Examples
+
+### Create a basic AppConfiguration resource
+
+```C# Snippet:AppConfigurationStoreFF
```

Comment:
Can we have some descriptions for this example?
I think we could fetch some information from the `[Description]` attribute on its corresponding test case, and the value of this attribute is a link, which should contain some descriptive information, we could summarize that and put it here.